bin/git-bc

```diff
@@ -20,10 +20,10 @@ if (opts.indexOf('-h') > -1) {
   gitBc.on('success', log.success);
 
   // Exits program execution on ESC keypress
-  process.stdin.on('keypress', function (ch, key) {
-    if (key && key.name === 'escape') {
+  process.stdin.on('keypress', function(ch, key) {
```

Comment:
Yeah, my bad for the space, also I was looking for autocomplete and thought if was more readable if we added multiples functions like autocomplete searches after to just separate on 2 lines the key verification but as this commit is only to add the Q key, i removed them :+1: